mindsdb/integrations/handlers/openbb_handler/openbb_handler.py

```diff
@@ -89,6 +89,7 @@ def connect(self) -> bool:
         """
         self.is_connected = False
         self.obb.account.login(pat=self.PAT)
+        # self.obb.account.refresh()
```

Comment:
Do we need this commented?

Reply:
it should be uncommented. 

it was by mistake, sorry